lock-api/src/main/java/com/palantir/lock/client/DialogueAdaptingConjureTimelockService.java

```diff
@@ -106,11 +110,24 @@ public ConjureRefreshLocksResponse refreshLocks(
         return dialogueDelegate.refreshLocks(authHeader, namespace, request);
     }
 
+    @Override
+    public ConjureRefreshLocksResponseV2 refreshLocksV2(
+            AuthHeader authHeader, String namespace, ConjureRefreshLocksRequestV2 request) {
+        throw new UnsupportedOperationException(
+                "This version of the AtlasDB client should not be using this " + "endpoint!");
```

Comment:
nit: concatenate strings

Reply:
Ah, fixed. Must be some weird combination of intellij and spotless